src/composables/useSearchParams.ts

```diff
@@ -1,10 +1,24 @@
 import { ref, onMounted, watch, watchEffect } from 'vue';import { z } from 'zod';
 import { useRoute, useRouter } from 'vue-router';
-import { flawFields, allowedEmptyFieldMapping } from '@/constants/flawFields';
+import { DateTime } from 'luxon';
+import { flawFields } from '@/constants/flawFields';
+
+export enum DateRange {
+  THIS_WEEK = 'This Week',
+  LAST_WEEK = 'Last Week',
+  THIS_MONTH = 'This Month',
+  LAST_MONTH = 'Last Month',
+  CUSTOM = 'Custom',
+}
```

Comment:
Since this is also used by the `DateRange` widget, we should move it to the constants folder